data/mods/Xedra_Evolved/items/ethereal_items.json

```diff
@@ -141,5 +141,22 @@
         "move_cost": 100
       }
     ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "constructed_hammer",
+    "name": { "str": "constructed hammer" },
+    "description": "A long handle end with two side head, one of which is hollow, and another… do not exist.  Weirdly, both heads are equally useful, and the balance of hammer is pretty good to be useful as a weapon.  By activating and attacking with the hollow part, the hammer creates a tremendous force, able to break a thick tree (or someone's spine) in half, but will be destroyed afterwards.",
```

Comment:
```suggestion
    "description": "A long handle end with two side head, one of which is hollow, and another… which doesn't exist.  Weirdly, both heads are equally useful, and the balance of hammer is pretty good to be useful as a weapon.  By activating and attacking with the hollow part, the hammer creates a tremendous force, able to break a thick tree (or someone's spine) in half, but will be destroyed afterwards.",
```

Reply:
The idea is here that second part do not exist - there is just nothing, you can move your hand freely - yey when you attack with it, it works like there is something.

Reply:
Yep, I love that idea.  I was trying to make it slightly clearer but I may have failed at that.